src/main/java/xyz/duncanruns/julti/gui/OptionsGUI.java

```diff
@@ -65,6 +65,7 @@ private void reloadComponents() {
         this.addComponentsOBS();
         this.addComponentsSound();
         this.addComponentsAffinity();
+        this.addComponentsLaunching();
```

Comment:
I might increase Julti Options window width if the new tab causes the tabs to collapse